packages/internet-header/cypress/e2e/meta-navigation.cy.ts

```diff
@@ -47,13 +47,13 @@ describe('meta-navigation', () => {
         cy.get('post-meta-navigation').should('not.exist');
       });
 
-      it(`sets css variable --meta-header-height to 0`, () => {
+      it(`sets css variable --meta-header-height to 3rem`, () => {
         cy.changeArg('meta', false);
         cy.get('swisspost-internet-header').within(() => {
           cy.get('header').then($el => {
             const style = getComputedStyle($el[0]);
             const metaHeaderHight = style.getPropertyValue('--meta-header-height');
-            expect(metaHeaderHight).to.eq('0px');
+            expect(metaHeaderHight).to.eq('3rem');
```

Comment:
@alizedebray not sure if it's the right fix, but I'm sure that you know.

Reply:
It seems the test was correct, only the name was wrong.

With a `meta="false"` attribute set on the `<swisspost-internet-header>`, the custom property should be set to 0px:
![grafik](https://github.com/swisspost/design-system/assets/33580481/33903297-a22b-4ca3-8069-886858ada6dc)

Reply:
Then something is wrong. Could you handle this PR?